local_planner/src/nodes/waypoint_generator.cpp

```diff
@@ -339,13 +339,23 @@ void WaypointGenerator::nextSmoothYaw(float dt) {
 }
 
 void WaypointGenerator::adaptSpeed() {
-  speed_ = planner_info_.cruise_velocity;
+
+  //lowpass filter the speed to get smoother accelerations
+  float lp_filter_constant = 0.1;
```

Comment:
Is this a value that is general enough that we can hard code in the code?

Reply:
I agree that having hard coded number is not ideal, but I don't think that we will ever need to touch that one again, so I don't think it makes sense to make it a parameter